src/components/Questionnaire/ManageQuestionnaireOrganizationsSheet.tsx

```diff
@@ -151,7 +151,9 @@ export default function ManageQuestionnaireOrganizationsSheet({
   const { t } = useTranslation();
   const [open, setOpen] = useState(false);
   const [searchQuery, setSearchQuery] = useState("");
-  const [selectedIds, setSelectedIds] = useState<string[]>([]);
+  const [selectedOrganizations, setSelectedOrganizations] = useState<
+    { id: string; name: string; description?: string }[]
+  >([]);
```

Comment:
import type instead !! (don't make direct commit from github you need to import the same in file)

```suggestion
  const [selectedOrganizations, setSelectedOrganizations] = useState<
    Organization[]
  >([]);
```

Reply:
Thanks for clarification. Just now I started making changes, but @Jacobjeevan already committed changes.